ci/utility/resources.yml

```diff
@@ -0,0 +1,31 @@
+resources:
+  - name: dataworks-s3-object-tagger
+    type: git
+    source:
+      branch: master
+      repository: dwp/dataworks-s3-object-tagger
+      uri: https://github.com/dwp/dataworks-s3-object-tagger.git
+    check_every: 720h
+    webhook_token: ((dataworks.concourse_github_webhook_token))
```

Comment:
you will need to raise a change to add the webhook to https://github.com/dwp/dataworks-github-config/blob/master/dataworks-s3-object-tagger.tf